### ISSUE TITLE
Avoid filing issues in this repo. File issues at: github.com/OpenCollective/opencollective/issues

### ISSUE THREAD

**Issue by asood123** [OpenCollective/opencollective-api#387]
File them at: http://github.com/OpenCollective/opencollective/issues

And don't close this issue.

**Comment by boneskull** [comment]
Why?  Creating issues referencing those issues in the commit message is very useful.

If we're having trouble managing multiple repositories, may I suggest a tool such as overv.io, waffle.io, or ZenHub...?

**Comment by sedubois** [comment]
You're trying to trigger Belgian sensibilities with your waffles ;)
Can these boards be publicly displayed from GitHub?

**Comment by asood123** [comment]
@boneskull We still link issues from commits cross-repo. We do lose the benefit of features like issues closing automatically, etc. Biggest reason to post this is that we don't look at issues on any other repository regularly. So, stuff filed here lingers and we appear non-responsive.

**Comment by boneskull** [comment]
@sedubois yes, the boards can be publicly displayed for overv.io for sure, and probably waffle.  ZenHub is just a browser extension as far as I remember.

> Biggest reason to post this is that we don't look at issues on any other repository regularly. So, stuff filed here lingers and we appear non-responsive.

A tool like I mentioned would mitigate this issue, as overv.io in particular is geared towards the "big picture" of managing multiple repositories at once.  

I'm mostly familiar with overv in this regard, but have heard good things about zenhub.  using one wouldn't completely absolve the need for the `OpenCollective` repo as we're using it--but we may be better served with a Trello-like tool for projects and issues that don't necessarily involve code.
